content/20.treatments.md

```diff
@@ -226,6 +226,16 @@ In principle, drugs that reduce the expression of these proteins or sterically h
 
 A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.
 
+Angiotensin-converting enzyme inhibitors and angiotensin II receptor blockers (ARBs) are among the most commonly prescribed medications, prescribed well over 100,000,000 times annually in the United States, for example.[@url:https://clincalc.com/DrugStats/Drugs/Lisinopril; @doi:10.1056/NEJMp1901657]
+Data from some animal models suggest several but not all ACE inhibitors and several ARBs increase ACE2 expression in cells of some organs.[@doi:10.1093/cvr/cvaa097]
+Clinical studies have not established that plasma ACE2 expression is increased in humans treated with these medications.[@doi:10.1093/europace/euw246]
+While randomized clinical trials are ongoing, a variety of observational studies have examined the relationship between exposure to ACE inhibitors or ARBs and outcomes in patients with COVID-19.
+Mehra and colleagues reported a 0.33 odds of death in hospitalized patients with COVID-19 who were treated with an ACE inhibitor at time of admission.[@doi:10.1056/NEJMoa2007621]
+In contrast, the authors did not find evidence of an association between exposure to an ARB at time of hospitalization and death.
+Because observational studies are subject to confounding, randomized controlled trials are the standard means of assessing the effects of medications, and the findings of this study cannot be interpreted as effects of a drug.
```

Comment:
```suggestion
Because observational studies are subject to confounding, randomized controlled trials are the standard means of assessing the effects of medications, and the findings of this study cannot be interpreted as indicating a protective effect of the drug.
```